boards/arm/nrf52810_pca10040/nrf52810_pca10040.dts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2017 Shawn Nock <shawn@monadnock.ca>
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52810.dtsi>
+
+/ {
+	model = "Nordic PCA10040 Dev Kit";
+	compatible = "nordic,pca10040-dk", "nordic,nrf52810-qfaa",
+		     "nordic,nrf52810";
+
+	chosen {
+		zephyr,console = &uart0;
+#ifdef CONFIG_MCUMGR_SMP_UART
+		zephyr,uart-mcumgr = &uart0;
+#endif
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+};
+
+&uart0 {
+	status = "ok";
+	compatible = "nordic,nrf-uarte";
```

Comment:
See above comment if this compat should just be in the SoC dtsi

Reply:
I removed this line from the board dts file, since it's already in the dtsi